sdk/eventhub/event-hubs/samples/typescript/src/iothubConnectionString.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/*
+  This sample demonstrates how to convert an Iot Hubs connection string to
```

Comment:
```suggestion
  This sample demonstrates how to convert an Iot Hub connection string to
```

Reply:
Thoughts on updating the text here to point to https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-messages-read-builtin, refer to "built in endpoint that uses Event Hubs", and that this sample converts the IotHub connection string to a connection string pointing to this "built in endpoint"?

Reply:
I like that since you could have other Event Hub endpoints that this would not convert into.